### ISSUE TITLE
Design: "Create a container" two button dialog doesn't follow design best practices

### ISSUE THREAD

**Issue by mairin** [containers/podman-desktop#2639]
TLDR; We need a UI design proposal that provides a better way to let users create a new container from either a containerfile or an existing image that doesn't use a two-button dialog like htis.

## The problems

This is the dialog that pops up when you click the "create a container" button on the Containers screen:
![Screenshot from 2023-05-25 11-41-05](https://github.com/containers/podman-desktop/assets/799683/807bd7f2-9935-4bf2-912b-008a284c752c)

- **There should be a clear primary / default action.** In our dialog - the two choices are kind of equal, there isn't really (in my understanding anyway :) )  a preferred or recommended way to create a container between these two choices, they are neutral. So we cannot really accent one as being a primary choice, so we give the users a harder decision - there's no default, if you don't want make a choice we offer it to the user - they have to choose. This goes against general best practices for UI design, and the effect is that it can overwhelm the user (by forcing them to read and making an assumption that adds a speedbump to their workflow). 

- **The primary action is normally on the right and is visible + designed to draw more attention.** We do actually have coloring in this dialog along the lines of indicating a primary choice, but the positioning I Think isn't correct. So I would expect the purple filled button "Container or Dockerfile" to be the primary since it's more visible/loud, but it is on the left. Typically, the primary button is on the right. 

- **The "Existing Image" button brings you to the Images screen without any context.** There's another issue I see with this dialog in that if I choose to create from Containerfile/Dockerfile, I get a form that guides me to do that, but if I choose to create a container from an existing image, it takes me to the Images screen without any context / explanation which feels weird. It's not a real flow, just a redirection.
